cmd/install/assets/hypershift-operator/hypershift.openshift.io_nodepools.yaml

```diff
@@ -197,6 +197,14 @@ spec:
                     required:
                     - instanceType
                     type: object
+                  type:
+                    description: PlatformType is a specific supported infrastructure provider.
+                    enum:
```

Comment:
where is this enum coming from? I can't see the validation kubebuilder tag for it in the API field.

Reply:
from here https://github.com/openshift/hypershift/blob/d15ea1f33e11c53a1a9124c98e8dadacf333234c/api/v1alpha1/hostedcluster_types.go#L119